File: Prenommer/Prenommer/Class2.vb
Public Class Class2

    Public Class Fich
        <JsonProperty("Title")>
        Public Property Titre As String ' En français

        <JsonProperty("Name")>
        Public Property Nom As String ' En français

        <JsonProperty("Charge")>
        Public Property Charge As String

        <JsonProperty("Institute")>
        Public Property Institut As String ' En français

        <JsonProperty("Celebration")>
        Public Property Celebration As String

        <JsonProperty("Birth")>
        Public Property Naissance As String ' En français

        <JsonProperty("Death")>
        Public Property Deces As String ' En français

        <JsonProperty("Otherparties")>
        Public Property Autresfetes As String ' En français

        <JsonProperty("Othernames")>
        Public Property Autresnoms As String ' En français

        <JsonProperty("Venerable")>
        Public Property Venerable As String

        <JsonProperty("Beatified")>
        Public Property Beatification As String ' En français

        <JsonProperty("Canonized")>
        Public Property Canonisation As String ' En français

        <JsonProperty("Heading")>
        Public Property TitrePrincipal As String ' En français

        <JsonProperty("Patron")>
        Public Property Patron As String

        <JsonProperty("Link")>
        Public Property Lien As String ' En français

        <JsonProperty("Image")>
        Public Property Image As String

        <JsonProperty("Biography")>
        Public Property Biographie As String ' En français

        <JsonProperty("Origin")>
        Public Property Origine As String ' En français

    End Class

    Public Class Fiche
        <JsonProperty("fiches")>
        Public Property IDUnique As String
        Public Property Nom As String
    End Class

    Public Class Prenommer
        <JsonProperty("fiches")>
        Public Property Fiches As Fiche()
    End Class

End Class

File: ProjetTest-main/Prenommer/Prenommer/Class2.cs
using Newtonsoft.Json;

namespace Prenommer
{

    public class Class2
    {

        public class Fich
        {

            [JsonProperty("title")]
            public string Title { get; set; }

            [JsonProperty("name")]
            public string Name { get; set; }

            [JsonProperty("charge")]
            public string Charge { get; set; }

            [JsonProperty("institute")]
            public string Institute { get; set; }

            [JsonProperty("celebration")]
            public string Celebration { get; set; }

            [JsonProperty("birth")]
            public string Birth { get; set; }

            [JsonProperty("death")]
            public string Death { get; set; }

            [JsonProperty("otherparties")]
            public string Otherparties { get; set; }

            [JsonProperty("othernames")]
            public string Othernames { get; set; }

            [JsonProperty("venerable")]
            public string Venerable { get; set; }

            [JsonProperty("beatified")]
            public string Beatified { get; set; }

            [JsonProperty("canonized")]
            public string Canonized { get; set; }

            [JsonProperty("heading")]
            public string Heading { get; set; }

            [JsonProperty("patron")]
            public string Patron { get; set; }

            [JsonProperty("link")]
            public string Link { get; set; }

            [JsonProperty("image")]
            public string Image { get; set; }

            [JsonProperty("biography")]
            public string Biography { get; set; }

            [JsonProperty("origin")]
            public string Origin { get; set; }

        }

        public class Prenommer
        {

            [JsonProperty("fiches")]
            public Fich[] Fiches { get; set; }

        }

    }
}